src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-right: 419px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.link_box {
+  margin-right: 20px;
+}
+
+.nav_link {
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  text-decoration: auto;
+  line-height: 60px;
+}
+
+.nav_list .link_box:last-child .nav_link {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav_link[data-qa='hover']:hover {
+  color: #00acdc;
+}
+
+@media (max-width: 1024px) {
+  .logo {
+    margin-right: 243px;
```

Comment:
The margin-right value for the `.logo` class in the media query seems arbitrary. Ensure that this value aligns with the design requirements for smaller screens.